CHANGES.md

```diff
@@ -17,6 +17,7 @@ Release Notes.
 * Add dubbo 3.x plugin.
 * Fix TracePathMatcher should match pattern "**" with paths end by "/"
 * Add support `returnedObj` expression for apm-customize-enhance-plugin
+* Enhance `returnedObj` expression to support Array, List and Map for apm-customize-enhance-plugin and `@Tag`
```

Comment:
I think we don't need this. Because this is enhancement of an unreleased feature.
But please make sure the doc is updated accordingly.

Reply:
CHANGES.md updated, and nothing need to be added to Customize-enhance-trace.md